src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -195,22 +181,33 @@ private IAsyncCollector<IStorageQueueMessage> BuildFromQueueAttribute(QueueAttri
             return new QueueAsyncCollector(queue, _messageEnqueuedWatcherGetter.Value);
         }
 
-        private static IStorageQueue GetQueue(QueueAttribute attrResolved)
+        private IStorageQueue GetQueue(QueueAttribute attrResolved)
```

Comment:
Won't fix. Refactoring all this to be async can be done in a separate change. I'm retaining existing behavior here.